src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -27,6 +28,7 @@
             <!-- Run after all custom normalizers -->
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
+        <service id="Symfony\Component\Serializer\Normalizer\ObjectNormalizer" alias="serializer.normalizer.object" public="false" />
```

Comment:
I'm not sure that there is a good use case to inject this normalizer using autowiring. I would not recommend to do it.